web-console/4.0-designs/overview/overview.md

```diff
@@ -0,0 +1,87 @@
+# Overview
+
+## Collapsed list view
+
+### General
+
+![list view](img/listview.png)
+* The project overview may be viewed in list view or topology view. It will default to show the list view. Users can toggle between the two views by using the view selector icons at the top right corner of the page.
+* Users may filter resources by name by using the filter bar in the top right.
+* The collapsed list view of the project overview will group items by app label (for example, `app=foo`) by default. Users may group by a different label by using the `Group by label` dropdown selector. The selector will be populated with existing labels.
+  * If no resources have labels, items will be shown in a regular list view listed alphabetically by resource type and then name and the `Group by label` dropdown will be hidden.
+* On hover, the row will be highlighted (#EDF8FF) and the icon fa-angle-right (pf-blue-400, #0088CE) will appear at the end of the row. When a row is selected (#DEF3FF), the side panel will open with a summary of that resource's overview, pushing the content over to the left.
+* If a user is not in the context of a selected project, the user will be shown an [empty state](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/empty-states/empty-states) until they select a project from the project dropdown.
+![empty state](img/ES_overview.png)
+
+### List columns
+
+* The first column will include resource badges and names. Users can redirect to a resource's details page by clicking the resource name.
+* The second column will display status updates and other relevant notifications and events such as crash looping containers and deployment failures. Users can hover on a status or notification to see more information.
+* The third column will be a summary of the item's metrics, including memory, CPU, and network. If metrics are not enabled, this column will be blank. If metrics are enabled but are not currently available, users will see "--" in place of the values.
+* The fourth column will show how many pods have deployed for that item. Users can click on the pods link to be directed to the resource's pods page.
+
+### Route URL
+
+![route](img/route.png)
+* Users may navigate to the application by clicking the route URL link (pf-blue-400, #0088CE) across from the application name. The icon fa-external-link (pf-blue-400, #0088CE) will be to the right of the URL to indicate that the link redirects users to an external location and opens a new browser tab.
+* Users may edit which route URL is displayed by clicking the black (pf-black-900, #292E34) pficon-edit icon. A popover will open where users may set their default route from a list of their routes. Users may also copy route URLs from this popover. Users may click Cancel or ‘x’ or click anywhere else on the screen to close the popover, or click Save to confirm their selection.
```

Comment:
I'm not sure how we can support this when you can pick different labels to group by. The routes under each group will change depending on the label. It seems to only work if we restrict group by to the `app` label.

Reply:
Hm I didn't know that could happen. Does it automatically choose the best route based on what you group by? Wondering then if this is all that useful as per your comment above. Thoughts @beanh66 ?

Reply:
The fundamental problem is that we're trying to show one thing when you could have many, and there's no good way to pick automatically.

This is what we did for the previous console: https://github.com/openshift/origin-web-console/pull/2887

Reply:
When we've met with customers, the sentiment seems to be that they want to see the full route on that main page and they want a way to decide which route is shown. The popover/modal design was well received so far. 

I would think users would group by some label and maybe stick to that view so only really need to set the default route once...but could be a very wrong assumption. I didn't realize the group by would affect routes either @bmignano. 

@spadgett we can try to investigate further on this and specifically call out that the route display can change based on grouping and see what we hear back. I have testing next Wednesday to get some feedback potentially.

Reply:
> I would think users would group by some label and maybe stick to that view so only really need to set the default route once...but could be a very wrong assumption.

We don't remember the selected label, so you would have to reselect every time you visit the overview (unless you leave it as the default app label).

> I didn't realize the group by would affect routes either @bmignano.

Yeah, it's a consideration outside of this default route design. If we show any route there, it would change when the groups change regardless.

> @spadgett we can try to investigate further on this and specifically call out that the route display can change based on grouping and see what we hear back. I have testing next Wednesday to get some feedback potentially.

We should also make it clear that it's only a personal setting for that one browser and other users won't see the default.

Reply:
It might be good to ask users as well about what we implemented in prior releases. We let users add an annotation to the route that marked it as the default overview route. This was visible to all users.